test/test_serre_green_naghdi_1d.jl

```diff
@@ -0,0 +1,68 @@
+module TestSerreGreenNaghdi1D
+
+using Test
+using DispersiveShallowWater
+
+include("test_util.jl")
+
+EXAMPLES_DIR = joinpath(examples_dir(), "serre_green_naghdi_1d")
+
+@testset "SerreGreenNaghdiEquations1D" begin
+    @trixi_testset "serre_green_naghdi_soliton.jl" begin
+        @test_trixi_include(joinpath(EXAMPLES_DIR,
+                                     "serre_green_naghdi_soliton.jl"),
+                            tspan=(0.0, 0.1),
+                            l2=[9.994998669268741e-7, 1.4703955789342244e-6, 0.0],
+                            linf=[6.5496216650196e-7, 1.027615853854691e-6, 0.0],
+                            cons_error=[0.0, 8.174581012099225e-10, 0.0],
+                            change_waterheight=0.0,
+                            change_entropy_modified=-3.1093350116861984e-11,
+                            atol=1e-9) # in order to make CI pass
```

Comment:
Did you check that we really need the `atol=1e-9` in all the tests?

Reply:
No, I didn't - just copy & paste. What's your experience - shall I just remove it and add it back if necessary or shall we try to find an "optimal" value?

Reply:
Yes, I would remove it and put it back if necessary. I usually had a look at the differences between the values I got locally (and thus put into the test files) and the values that came from CI and then chose the tolerance to be the smallest value, which is a power of ten for which all tests pass.

Reply:
CI passes with strict tolerances 🥳

Reply:
That's nice, but why did the test values change in [53a172d](https://github.com/JoshuaLampert/DispersiveShallowWater.jl/pull/127/commits/53a172d90f852befc07b0c4b7484f9935ad4465c)? Are the current values the ones you get locally?

Reply:
Yes, the current values are what I get locally

Reply:
Maybe there was some copy-paste issue earlier or some the implementation changed slightly after writing the tests? 🤷

Reply:
Alright :+1: